includes/js/config.js

```diff
@@ -11,5 +11,12 @@ var config = {
 	canvasHeight: 800,
 
 	tileSize: 40,
-	spriteSheet: "spritesheet.png"
+	spriteSheet: "spritesheet.png",
+
+	key_left: 37,
+	key_up: 38,
+	key_right: 39,
+	key_down: 40,
+	key_use: 13,
+	key_shift: 16
```

Comment:
All other configuration variables are camelCase, it would be a good custom to keep it so.
If you want more clarity, you could create keys: {} objects and list the keys under there.